Pod/Classes/PINURLSessionManager.m

```diff
@@ -42,10 +44,11 @@ - (void)invalidateSessionAndCancelTasks
 - (NSURLSessionDataTask *)dataTaskWithRequest:(NSURLRequest *)request completionHandler:(void (^)(NSURLResponse *response, NSError *error))completionHandler
 {
     [self lock];
-    NSURLSessionDataTask *dataTask = [self.session dataTaskWithRequest:request];
-    if (completionHandler) {
-        [self.completions setObject:completionHandler forKey:@(dataTask.taskIdentifier)];
-    }
+        NSURLSessionDataTask *dataTask = [self.session dataTaskWithRequest:request];
+        if (completionHandler) {
+            [self.completions setObject:completionHandler forKey:@(dataTask.taskIdentifier)];
+        }
+        [self.delegateQueues setObject:dispatch_queue_create([[NSString stringWithFormat:@"PINURLSessionManager delegate queue - %ld", (unsigned long)dataTask.taskIdentifier] UTF8String], DISPATCH_QUEUE_SERIAL) forKey:@(dataTask.taskIdentifier)];
```

Comment:
This would be a little more readable if you create the dispatch queue as a local variable.